DOCUMENTATION.md

```diff
@@ -1686,6 +1686,10 @@ The following UI controls are available:
   * [VRTK_Lever](#vrtk_lever)
   * [VRTK_Slider](#vrtk_slider)
 
+All controls emit the following events:
```

Comment:
I don't think we should copy paste it. What I could imagine is some sort of back reference stating "In addition to the default control events and methods, the following are exposed in addition:".